### ISSUE TITLE
Getting a critical alert on this widget from Wordfence

### ISSUE THREAD

**Issue by jenryland** [scottsweb/wp-instagram-widget#115]
Alert says "The Plugin "WP Instagram Widget" has been removed from wordpress.org.It may have compatibility problems with the current version of WordPress or unknown security issues. Get more information.
Plugin URL: https://github.com/scottsweb/wp-instagram-widget"

**Comment by scottsweb** [comment]
The plugin was removed from the .org repo as Facebook/Instagram made a trademark complaint. There is not a great deal I can do unfortunately.

**Comment by jenryland** [comment]
Thanks - sorry to hear that!

**Comment by DeliciousThemes** [comment]
Hey @scottsweb , does the plugin have a future? I want to include it in one of my themes via TGMPA. Maybe you would want to rebrand it into something which align to Fb/Insta terms? What are your plans? 

Thanks!

**Comment by scottsweb** [comment]
No plans to get it back on the .org repo. The code will remain here... Will be interesting to see if the same issue arises on GitHub.

**Comment by DeliciousThemes** [comment]
I totally understand! 
So what was all about? Was the branding/name that got it down or the code and the way the plugin gets the data from Insta(without an API key, all that authentication stuff)?

**Comment by scottsweb** [comment]
This time it was a trademark issue. The name and use of their old logo in the plugin header. The plugin was also removed earlier in the year also for scraping, rather than using the API. I managed to fight that one though. 

Bottom line, I have always maintained these plugins in my own time and any barrier to making that happen will just push me in a different direction. I would much rather spend energy on something productive and more interesting. There is a second issue, I don't use Instagram or Facebook and I would rather other people stopped too, so I shouldn't really be encouraging it through this plugin. It is crazy to me to think a company would go after an open source plugin that provides that very company with exposure.
